master/internal/scheduler/fitting.go

```diff
@@ -92,7 +93,7 @@ func isViable(task *Task, agent *agentState, constraints ...HardConstraint) bool
 	return true
 }
 
-func findMultiSlotFits(
+func findDedicatedAgentFits(
```

Comment:
non-blocking: don't think this will happen in our current implementation but we should add a check here to avoid having zero slot tasks going into this function.